sshd_config

```diff
@@ -3,7 +3,7 @@ HostKey /.bottlerocket/host-containers/admin/etc/ssh/ssh_host_ecdsa_key
 HostKey /.bottlerocket/host-containers/admin/etc/ssh/ssh_host_ed25519_key
 
 PasswordAuthentication no
-
+TrustedUserCAKeys /home/ec2-user/.ssh/authorized_keys
```

Comment:
With `AuthorizedKeysFile`, this can be a path relative to the home directory of the user logging in, and that's the form used by default (`.ssh/authorized_keys`).  The man page isn't clear, but I suspect `TrustedUserCAKeys` should work the same way, and then we wouldn't have to hardcode the username.  Do you know if that works here?  On other platforms the user might not be ec2-user, so it'd be nice to avoid specifying when possible, easing the clean-up later.

I also think it'd be a good idea to explicitly list `AuthorizedKeysFile`, even though it's the default, to make it clear that we expect to accept both.

So, something like this, if it works:  (not sure on the best comment wording)
```
PasswordAuthentication no
ChallengeResponseAuthentication no

# Allow authentication with standard key pairs or certificates.
AuthorizedKeysFile .ssh/authorized_keys
TrustedUserCAKeys .ssh/authorized_keys
```

Reply:
@tjkirch tested this out locally - AuthorizedKeysFile does work as a relative path as the docs indicate, but TrustedUserCAKeys does not use the path as a relative path. Verified this by first testing with `TrustedUserCAKeys .ssh/authorized_keys`, which did not work. Copying the file in the container to `/.ssh/authorized_keys` allowed me to login with my signed certificate - and then removing that copied file again denied me access.